mne/beamformer/_compute_beamformer.py

```diff
@@ -116,7 +116,10 @@ def _compare_ch_names(names1, names2, bads):
 
 def _check_one_ch_type(info, picks, noise_cov):
     """Check number of sensor types and presence of noise covariance matrix."""
+    comps = info['comps']
+    info['comps'] = []
     info_pick = pick_info(info, sel=picks)
+    info['comps'] = comps
```

Comment:
@larsoner do you have a nicer suggestion?

Reply:
For now maybe just put a `# XXX this is a hack that should be refactored` until we converge and tackle #5342